### DISCUSSION TITLE
certified pimcore developer

### DISCUSSION THREAD

**Discussion by BjBue** [pimcore/pimcore#14904] [General]
Hello,
i am a little sad.
Is it right that i get better support for developing pimcore and applications using pimcore when i have done the certificates?
Thank you for response.

**Comment by solverat** [comment]
There is no reason to be sad.

The certificate will
- improve your PIMCORE Knowledge 
- help you to stay updated in dev context
- support open source (nothing is for free)
- hand you a certificate :) sometimes that's good if your in a pitch for a new client

The certificate will not:
- give you better support. Why should it? You could checkout the partner program which comes with some extra support, but this is still limited
- transform you into a first class PIMCORE developer. You need at least some basic Symfony Knowledge (and ExtJs, if you want do extend the admin backend)
